app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffectHandler.kt

```diff
@@ -27,6 +30,7 @@ class ChangeLanguageEffectHandler @AssistedInject constructor(
         .addTransformer(UpdateCurrentLanguageEffect::class.java, updateCurrentLanguage(settingsRepository, schedulersProvider.io()))
         .addAction(GoBack::class.java, uiActions::goBackToPreviousScreen, schedulersProvider.ui())
         .addAction(RestartActivity::class.java, uiActions::restartActivity, schedulersProvider.ui())
+        .addAction(TriggerSync::class.java, { dataSync.fireAndForgetSync(DAILY) }, schedulersProvider.io())
```

Comment:
We don't need to run this on any specific scheduler since `DataSync` takes care of this internally.

Reply:
Oh yeah, forgot about that 😅

Reply:
This still seems to be running on the `.io()` scheduler.

Reply:
fixed